### PR TITLE
sugar-os-rpi v1

### PR DESCRIPTION
**Pull request by 44yu5h**
- 1.5GB
- RPi Lite OS + Sugar shell
- Activities : default ones + tower-of-hanoi, number-guessing, flappy (by Vaibhav), turtleart-activity, and rpi_control_center
- Temporary fix for the browse-activity issue (remove v207 that comes from apt, install v208, install dependencies)

### CODE REVIEW

**Comment by 44yu5h** [comment]
@chimosky ping!

Let me know if there's something to change

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+The parent repository is [pi-gen](https://github.com/RPi-Distro/pi-gen/) and history is cleaned at commit `dca93d2` to have a clean start.
```
Comment (chimosky) [inline README.md]: This is no longer the case, why is it still here?

**Comment by chimosky** [comment]
What's the need for the `pigen-readme.md` file?

Avoid creating PRs from your master branch, checkout the first part of our [contributing docs](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#contributing).

**Comment by 44yu5h** [comment]
> What's the need for the `pigen-readme.md` file?

I imagined it would be easier to refer to the instructions quickly to understand the code. 

> Avoid creating PRs from your master branch, checkout the first part of our [contributing docs](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#contributing).

Thanks for letting me know. Is there any implication for this or is it just a standard practice? 
I'll create a new pr

**Comment by chimosky** [comment]
> Thanks for letting me know. Is there any implication for this or is it just a standard practice?

It's standard practice when using the pull request model, it helps group commits that's particular to the changes you're making, which means you can just keep pushing changes to that particular branch as you make them and they'll be part of the PR. 

You might find GH docs on [pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) helpful.

**Comment by 44yu5h** [comment]
Ah got it. Thanks.

**Pull request closed without merging**